test/e2e/test-apps/sessions/abnormal-exit/src/main.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+
+const { app, BrowserWindow } = require('electron');
+const { init } = require('@sentry/electron');
+
+init({
+  dsn: '__DSN__',
+  debug: true,
+  onFatalError: () => {},
+});
+
+app.on('ready', () => {
+  const mainWindow = new BrowserWindow({
+    show: false,
+    webPreferences: {
+      nodeIntegration: true,
+      contextIsolation: false,
```

Comment:
Should this be configurable?

Reply:
This is an integration test app. Most of the tests are run with `nodeIntegration: true, contextIsolation: false` because otherwise we need to bundle the renderer code due to lack of `require`.